### PR TITLE
Fix some Python 3.8 issues

### PR DESCRIPTION
**Pull request by vstinner**
* tox.ini: add py38
* tests.tools: add Constant() helper

Partially addresses #28

### CODE REVIEW

File: tests/tools.py [hunk 1]
```diff
@@ -132,3 +132,14 @@ def verify_all_nodes(self, test_case):
       tested_nodes += 1
 
     return tested_nodes
+
+
+def Constant(name):
```
Comment (dsagal) [inline tests/tools.py]: What do you think of changing `MarkChecker.view_node` to replace `node.__class__.__name__` with `get_node_name(node)` which is something like this:
```
def get_node_name(node):
   name = node.__class__.__name__
   return "Constant" if name in ("Num", "Str", "NameConstant") else name
```
Then we can change code throughout to expect things like `Constant:17` or `Constant:'hello'`, which seems a bit simpler.

**Comment by vstinner** [comment]
Sorry, I don't have the bandwidth to update my PR. If someone wants to take it over, please go ahead :-)

**Comment by keszybz** [comment]
See #30.

**Comment by keszybz** [comment]
This can be closed now.

**Pull request closed without merging**